tensorboard/webapp/settings/dialog_component.ts

```diff
@@ -59,18 +81,45 @@ const getReloadPeriodInSec = createSelector(
         Reload period has to be minimum of 15 seconds.
       </mat-error>
     </div>
+    <div>
+      <mat-form-field>
+        <input
+          class="page-size"
+          matInput
+          type="number"
+          placeholder="Pagination Limit"
+          [formControl]="paginationControl"
+        />
+      </mat-form-field>
+      <mat-error
+        *ngIf="
+          paginationControl.hasError('min') ||
+          paginationControl.hasError('required') ||
+          paginationControl.hasError('integer')
```

Comment:
Does `paginationControl.invalid` suffice?

Reply:
Yes, it does! Used.